src/style.css

```diff
@@ -1 +1,73 @@
-/* Styles go here */
+body {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+}
+
+:root {
```

Comment:
this entry is implicit, because it is not clear to which class or tag it is applied, it is better to apply it to html